wordpress-mobile/AztecEditor-Android
# First class support for strong

### Fix #709 

This PR:
1. Allows the `<strong>` element to be preserved when changing from html mode to visual and back
2. Makes the `<strong>` HTML element the "styling" element used when tapping on the "B" toolbar button

This implementation will probably help us also include similar support for the `em` element.

Implementation details:
* `ToolbarAction` now includes a set of formats it supports. What that means is that, for example, the corresponding toolbar button will get activated whenever any of those formats are detected. But, only the first format is considered "primary" and is the one applied when user taps the toolbar button to insert the format.
* The formats in the list above are considered "same". That is, the `isSameInlineSpanType` method will return `true` for them. That's useful for the cases we need to expand one into the other. For example, the [testExpandBoldAndSetToStrongFormatting](https://github.com/wordpress-mobile/AztecEditor-Android/blob/c4367d722f698e794ca4516a59203ed2544be5b2/app/src/androidTest/kotlin/org/wordpress/aztec/demo/tests/ToolbarHighlightingTests.kt#L109) where `bold` is already in the text but gets transformed into `strong` when the style is applied to the whole text.


### Test
1. Load `<strong>this is strong</strong> and <b>this is bold</b>` into the demo app in html mode
2. Switch to visual mode
3. Switch back to html mode and notice that the `strong`s are still there, and same goes for `b`s

### Review
@mzorz or @daniloercoli 🙇‍♂️
